jcl/src/java.base/share/classes/openj9/internal/tools/attach/target/AttachHandler.java

```diff
@@ -23,6 +23,8 @@
 
 package openj9.internal.tools.attach.target;
 
+import com.ibm.oti.vm.VM;
+
```

Comment:
Please leave this import where it was (static and non-static imports should be grouped separately).

Reply:
Moved it after static imports.